t/URT/t/52_limit_cache_size.t

```diff
@@ -156,7 +161,7 @@ sub count_things_in_cache {
 #        foreach (values %{$UR::Context::all_objects_loaded->{$c}} ) {
 #            print "\tid ",$_->id,"\n";
 #        }
-        $count += scalar(values %{$UR::Context::all_objects_loaded->{$c}});
+        $count += scalar(grep { defined } values %{$UR::Context::all_objects_loaded->{$c}});
```

Comment:
I asked because I didn't see how the test covered the change.  However, I checked out the PR and I see that the test fails by crashing under the old code which I guess should have been obvious.